x/liquidity/keeper/batch.go

```diff
@@ -126,15 +126,15 @@ func (k Keeper) ExecutePoolBatch(ctx sdk.Context) {
 	})
 }
 
-// In order to deal with the batch at once, the coins of msgs deposited in escrow.
+// In order to deal with the batch at the same time, the coins of msgs are deposited in escrow.
```

Comment:
instead of `once` which is often incorrectly used as `after` and is correctly assumed to precede `twice`, let's instead use `at the same time`